app/assets/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -108,7 +106,7 @@ class DatasetInfoTabView extends Component {
 
 function mapStateToProps(state: OxalisState): DatasetInfoTabProps {
   return {
-    skeletonTracing: enforceSkeletonTracing(state.tracing),
```

Comment:
Sorry about that. I introduced that change. The fact that there was a "skeletonTracing" prop led me to believe that this view will only be rendered if there is a skeleton tracing. I should have double checked that.